include/zephyr/sensing/sensing.h

```diff
@@ -150,7 +152,7 @@ struct sensing_sensor_info {
  */
 struct sensing_callback_list {
 	sensing_data_event_t on_data_event;
-	void *context;
+	void *context; /**< Associated context with on_data_event */
```

Comment:
Thanks @lixuzha! When pointing out the missing doxygen comments in the recently added fields, I was also hoping you could do a full pass on the various sensing headers :) For example, it probably doesn't make sense to only document `context` in this struct, and not `on_data_event` at the line just above.

If this helps, you may check https://docs.zephyrproject.org/api-coverage/latest/zephyr/sensing/index.html

Reply:
Fixed. Thanks for @qianruh 's help.